src/widgets/refinement-list/refinement-list.js

```diff
@@ -134,10 +134,11 @@ function refinementList({
       return createURL(state.toggleRefinement(attributeName, facetValue));
     }
 
-    // Pass count of currently selected items to the header template
+    // Pass count of total and currently selected items to the header template
+    const facetsCount = facetValues.length;
```

Comment:
Sure.
My use-case was for an output in the header like so:
**Attribute Name** _(3/20)_

...where _3/20_ is selected `refinedFacetsCount`/`totalFacetsCount`